### PR TITLE
IGNITE-9017 clear-cache-stats

### PR DESCRIPTION
**Pull request by gurustron**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/processors/platform/cluster/PlatformClusterGroup.java [hunk 1]
```diff
@@ -638,4 +638,24 @@ private void writeDataStorageMetrics(BinaryRawWriter writer, DataStorageMetrics
...
+
+        Collection<String> cacheNames = new ArrayList<>(cnt);
+
+        for (int i = 0; i < cnt; i++) {
+            cacheNames.add(reader.readString());
+        }
+
+        return cacheNames;
+    }
+
```
Comment (ptupitsyn) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/platform/cluster/PlatformClusterGroup.java]: Redundant blank line

File: modules/platforms/dotnet/Apache.Ignite.Core.Tests/Cache/CacheMetricsTest.cs [hunk 2]
```diff
@@ -180,6 +180,58 @@ public void TestClusterGroupMetrics()
...
+        /// Test clearing cache metrics
+        /// </summary>
+        [Test]
+        public void TestCacheClearStatistics()
+        {
+            TestClearStatistics("TestCacheClearStatistics", cache => cache.ClearStatistics());
+        }
+
+        /// <summary>
+        /// Tests the cache metrics enable/disable for cluster group
```
Comment (ptupitsyn) [inline modules/platforms/dotnet/Apache.Ignite.Core.Tests/Cache/CacheMetricsTest.cs]: Misleading comment

File: modules/platforms/dotnet/Apache.Ignite.Core/Cluster/IClusterGroup.cs [hunk 3]
```diff
@@ -259,5 +259,12 @@ public interface IClusterGroup
         /// <param name="enabled">Enabled flag</param>
         [SuppressMessage("Microsoft.Design", "CA1024:UsePropertiesWhereAppropriate", Justification = "Semantics.")]
         void EnableStatistics(IEnumerable<string> cacheNames, bool enabled);
+
+        /// <summary>
+        /// Clears statistics for caches cluster wide.
+        /// </summary>
+        /// <param name="caches">Collection of cache names.</param>
+        [SuppressMessage("Microsoft.Design", "CA1024:UsePropertiesWhereAppropriate", Justification = "Semantics.")]
```
Comment (ptupitsyn) [inline modules/platforms/dotnet/Apache.Ignite.Core/Cluster/IClusterGroup.cs]: Do we really need this suppression? Is there a warning? Same for the method above.

File: modules/platforms/dotnet/Apache.Ignite.Core.Tests/Cache/CacheMetricsTest.cs (hunk 2 above)
Comment (gurustron) [reply modules/platforms/dotnet/Apache.Ignite.Core.Tests/Cache/CacheMetricsTest.cs]: fixed

**Pull request merged**